ccd-definition/CaseTypeTab.json

```diff
@@ -20,7 +20,7 @@
     "TabFieldDisplayOrder": 2
   },
   {
-    "LiveFrom": 43101,
+    "LiveFrom": "01/01/2018",
```

Comment:
Can we please keep all dates the same? I do not care much what date we decide to use, just would like to have same start date everywhere.

Reply:
OK changed